tooling/nargo_cli/build.rs

```diff
@@ -217,7 +218,6 @@ fn test_{test_name}(force_brillig: ForceBrillig, inliner_aggressiveness: Inliner
         nargo.arg("50");
 
         // Check whether the test case is non-deterministic
```

Comment:
Comment did not move 👆

Reply:
🤦 updated here https://github.com/noir-lang/noir/pull/6959